resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
```

Comment:
Sorry, that wasn't very clear on my part.  So here's an example of a change that we'd need to make in the Puppet Server repo in order to opt-in to the "restart file" pattern:

https://github.com/puppetlabs/puppetserver/pull/1160/files#diff-f5caea6fde69fc958f8bb5d6f8605f97R5

We have to get that path into our configuration.  But then we also need to get the path into these start/reload scripts.  In the current implementation, we are duplicating the `/opt/puppetlabs/server/data` prefix here in the EZBake repo and there, in the Puppet Server repo.  And we'd basically be expecting/requiring any other consumers of this functionality to do it the same way.

So, the other option that I thought of was to add a new EZBake config option called `restart-file`, which would go somewhere like here:

https://github.com/puppetlabs/puppetserver/blob/master/project.clj#L87-L95

In that case, the line above in this PR would just get the full path from `EZBake::Config`, and the `/opt/puppetlabs/server/data` part wouldn't need to appear in this PR.  We'd still have the path hard-coded in two places, but they'd both be in the Puppet Server repo and there wouldn't be anything in ezbake that forced people to choose a specific path.

The more I think about it, the more I think that that is probably a better way to go.  Let me know if you agree, and we can re-work this PR.